htextview-evaporate/src/main/java/com/hanks/htextview/evaporate/EvaporateText.java

```diff
@@ -57,19 +62,19 @@ public void onAnimationUpdate(ValueAnimator animation) {
         duration = (long) (charTime + charTime / mostCount * (n - 1));
     }
 
+
     @Override
-    public void animateText(final CharSequence text) {
```

Comment:
Hi @tearfulDalvik ,
may I ask why did you change animateText method? Did you find a case where old implementation is not working?

Reply:
oops. I pushed an old version of code. Check this now please.